`send_and_wait` doesn't properly mock `aiokafka.send_and_wait`

**Describe the bug**

aiokafka's version of `send_and_wait` has 2 positional arguments (see https://aiokafka.readthedocs.io/en/stable/api.html#aiokafka.AIOKafkaProducer.send_and_wait). However, in `FakeAIOKafkaProducer` these arguments aren't there and the *args and **kwargs are not properly passed from `send_and_wait` to `_produce` and you get an exception when using this function:

`TypeError: FakeAIOKafkaProducer._produce() missing 1 required positional argument: 'topic'`

**To Reproduce**
Steps to reproduce the behavior:
```
producer = FakeAIOKafkaProducer()
producer.send_and_wait("topic_name", "sdfjhasdfhjsa", key="datakey")
```
:boom: 

**Expected behavior**
It should not throw an exception

**Screenshots**
```
/usr/local/lib/python3.12/site-packages/sentry_sdk/tracing_utils_py3.py:64: in func_with_tracing
    return func(*args, **kwargs)
src/kafka.py:90: in process
    asyncio.run(self.send_updated_records(plant_data.plant_key, updated_data))
/usr/local/lib/python3.12/asyncio/runners.py:194: in run
    return runner.run(main)
/usr/local/lib/python3.12/asyncio/runners.py:118: in run
    return self._loop.run_until_complete(task)
/usr/local/lib/python3.12/asyncio/base_events.py:685: in run_until_complete
    return future.result()
src/kafka.py:106: in send_updated_records
    await self.producer.send_and_wait(ct, serialized_record, key=plant_key)
/usr/local/lib/python3.12/site-packages/mockafka/aiokafka/aiokafka_producer.py:40: in send_and_wait
    await self.send()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <mockafka.aiokafka.aiokafka_producer.FakeAIOKafkaProducer object at 0x7f5302d6dd30>, args = ()
kwargs = {}

    async def send(self, *args, **kwargs):
>       await self._produce(**kwargs)
E       TypeError: FakeAIOKafkaProducer._produce() missing 1 required positional argument: 'topic'
```

**Desktop (please complete the following information):**
 - OS: linux
 - python version: 3.12
 - mockafka-py version: 0.1.52

--- a/mockafka/aiokafka/aiokafka_producer.py
+++ b/mockafka/aiokafka/aiokafka_producer.py
@@ -22,19 +22,47 @@ class FakeAIOKafkaProducer:
     - send_and_wait(): Call send().
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
         self.kafka = KafkaStore()
 
-    async def _produce(self, topic, value=None, *args, **kwargs):
+    async def _produce(self, topic, value=None, *args, **kwargs) -> None:
         # create a message and call produce kafka
         message = Message(value=value, topic=topic, *args, **kwargs)
         self.kafka.produce(message=message, topic=topic, partition=kwargs['partition'])
 
-    async def start(self):
+    async def start(self) -> None:
         pass
 
-    async def send(self, *args, **kwargs):
-        await self._produce(**kwargs)
+    async def send(self,
+                   topic,
+                   value=None,
+                   key=None,
+                   partition=0,
+                   timestamp_ms=None,
+                   headers=None,
+                   ) -> None:
+        await self._produce(
+            topic=topic,
+            value=value,
+            key=key,
+            partition=partition,
+            headers=headers,
+            timestamp_ms=timestamp_ms,
+        )
 
-    async def send_and_wait(self, *args, **kwargs):
-        await self.send(**kwargs)
+    async def send_and_wait(self,
+                            topic,
+                            value=None,
+                            key=None,
+                            partition=0,
+                            timestamp_ms=None,
+                            headers=None,
+                            ) -> None:
+        await self._produce(
+            topic=topic,
+            value=value,
+            key=key,
+            partition=partition,
+            headers=headers,
+            timestamp_ms=timestamp_ms,
+        )

--- a/mockafka/aiokafka/aiokafka_producer.py
+++ b/mockafka/aiokafka/aiokafka_producer.py
@@ -37,4 +37,4 @@ class FakeAIOKafkaProducer:
         await self._produce(**kwargs)
 
     async def send_and_wait(self, *args, **kwargs):
-        await self.send()
+        await self.send(**kwargs)

--- a/tests/test_aiokafka/test_aiokafka_producer.py
+++ b/tests/test_aiokafka/test_aiokafka_producer.py
@@ -20,6 +20,7 @@ class TestFakeProducer(IsolatedAsyncioTestCase):
         await self.admin_client.create_topics(new_topics=[
             NewTopic(name='test1', num_partitions=4, replication_factor=1),
             NewTopic(name='test2', num_partitions=8, replication_factor=1),
+            NewTopic(name='topic_test', num_partitions=8, replication_factor=1),
         ])
 
     @pytest.fixture(autouse=True)
@@ -64,3 +65,10 @@ class TestFakeProducer(IsolatedAsyncioTestCase):
         self.assertEqual(message.headers(), {})
         self.assertEqual(message.error(), None)
         self.assertEqual(message.latency(), None)
+
+    async def test_send_and_wait(self):
+        await self._create_mock_topic()
+        await self.producer.send_and_wait("topic_test", "sdfjhasdfhjsa", key="datakey")
+        message: Message = self.kafka.get_messages_in_partition(topic="topic_test", partition=0)[0]
+        self.assertEqual(message.key(), "datakey")
+        self.assertEqual(message.topic(), "topic_test")

